android/src/main/java/io/mosip/tuvali/wallet/Wallet.kt

```diff
@@ -45,7 +48,9 @@ class Wallet(
   private val handlerThread = HandlerThread("TransferHandlerThread", Process.THREAD_PRIORITY_DEFAULT)
 
   private var central: Central
-  private val maxMTU = 185
+
+  private var negotiatedMTU = 100
+  private val mtuValues = arrayOf(512, 185, 100)
```

Comment:
(nit: naming) these aren't MTU values specifically, rather they are maxChunkSize values, do you also feel the same way due to how Android's BLE framework returns MTU(with the 3bytes of header)?

Reply:
This is actual MTU size not the maxChunk value. We are not using the last 5 bytes just to avoid edge cases.